src/server/package/src/model_explorer/pytorch_exported_program_adater_impl.py

```diff
@@ -177,7 +179,12 @@ def add_node_attrs(self, fx_node: torch.fx.node.Node, node: GraphNode):
       if tensor_spec:
         node.attrs.append(KeyValue(key='target', value=str(tensor_spec[0])))
         node.attrs.append(
-            KeyValue(key='__value', value=self.print_tensor(tensor_spec[1]))
+            KeyValue(
+                key='__value',
+                value=self.print_tensor(
+                    tensor_spec[1], self.settings['const_element_count_limit']
```

Comment:
I think we need to check the existence of `const_element_count_limit`. If it doesn't exist, we should use the default value (16)

Reply:
Done!